lib/line/bot/httpclient.rb

```diff
@@ -38,10 +38,8 @@ def http(uri)
           http.use_ssl = true
         end
 
-        if http_options
-          http_options.each do |key, value|
-            http.send("#{key}=", value)
-          end
+        http_options&.each do |key, value|
```

Comment:
Rubocop encourages to use safe navigation operator. It is introduced in ruby 2.3.

```
lib/line/bot/httpclient.rb:41:9: C: Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.
        if http_options ...
        ^^^^^^^^^^^^^^^
```